0.19.0-release-notes.md

```diff
@@ -9,6 +9,8 @@ This is the draft release notes for 0.19.0, scheduled to be released around
 -   Symbolic links are now always treated as ordinary files by the global
     modifiers `type:dir` and `type:regular` in wildcard expansions.
 
+-   Support for shared vars has been removed; e.g., `store:shared-var`.
```

Comment:
I know "e.g.,` is the preferred style in US English, but Elvish's documentation never uses a comma after "e.g." or "i.e.". I'll remove that.

Reply:
It's probably best just to avoid *i.e.* and *e.g.*, as suggested by https://developers.google.com/style/abbreviations#dont-use suggests.

Reply:
> It's probably best just to avoid _i.e._ and _e.g._, as suggested by https://developers.google.com/style/abbreviations#dont-use suggests.

Good point. Those abbreviations are strongly English (Anglo Saxon/Latin) specific and may not be recognized by  individuals for whom English is a second language. However, note that there are a huge number of such uses in this project -- 257 lines by my count. What do you want to do about them? I'm happy to make a "janitor" change that replaces them and also augment the *CONTRIBUTING.md* file to make note of this project's preference.